pages/library/[libGameId]/index.jsx

```diff
@@ -31,11 +80,45 @@ export default function LibraryGameDetails() {
           <BackButton href="/library" />
           <h1>{data[0].name}</h1>
         </StyledContent>
-      </StyledContainer>
+      </StyledImageContainer>
 
-      <StyledArticle>
-        <StyledDescription>{data[0].summary}</StyledDescription>
-      </StyledArticle>
+      <StyledContainer>
+        <StyledArticle>
+          <StyledDescription>{data[0].summary}</StyledDescription>
+        </StyledArticle>
+        {gameExists(data) && buttonMode !== "success" && (
```

Comment:
Once you have a variable where you store the single game object, you can pass this variable here:

```suggestion
        {gameExists(singleGame) && buttonMode !== "success" && (
```